src/scripts/config/mlz_config_create.sh

```diff
@@ -9,8 +9,6 @@
 #
 # Create MLZ backend config resources
 
-set -e
```

Comment:
We should be exiting on error

Reply:
Lets discuss so I can better explain what I am seeing in the code